app/Http/Controllers/Mship/Feedback.php

```diff
@@ -40,8 +41,13 @@ public function postFeedbackFormSelect(Request $request)
         return Redirect::route('mship.feedback.new.form', [$request->input('feedback_type')]);
     }
 
-    public function getFeedback(Form $form)
+    public function getFeedback(Form $form, Request $request)
     {
+        // Get a list of "pre-fillable" questions from the request, based on question slug
+        // To allow direct links from e.g euroscope profiles via vats.im/atcfb?usercid=12345
+        $preFillable = $request->only(['usercid']);
```

Comment:
Minor point: I'd ask to keep the nomenclature for 'user' inline with the Core naming for an account.

```suggestion
        $preFillable = $request->only(['accountcid']);
```

Or just use `cid` as the query parameter.

Reply:
Have swapped to `cid` as it's shorter